### PR TITLE
[TRANSLATION] add and update indonesian apps translation

### PR DESCRIPTION
**Pull request by Mas4hmad**
## Purpose

Add and indonesian language for several apps

## What is Changed?
- USETUP (add translation)
- HOTPLUG (add translation)
- CONSRV  (update translation)
- USERSRV (update translation)
- USER32  (update translation)
- ACLUI  (update translation)
- LICCPA  (update translation)
- SUBST  (update translation)
- REGSRV32  (update translation)
- REGEDIT-CLB  (update translation)
- CMDUTILS  (update translation)
- ACPPAGE  (update translation)
- TIMEDATE  (update translation)
- SOLITAIRE  (update translation)
- FDEBUG  (update translation)
- SYSSETUP (update translation)

### CODE REVIEW

**Comment by Mas4hmad** [comment]
thanks, will do...
but there are several problem with my knowledge about "ammend", "Rebase", "pick" and "HEAD"

**Comment by binarymaster** [comment]
You can just set your full name `Mas Ahmad Muhammad` in your [profile settings](https://github.com/settings/profile). Once PR is merged, it will have correct author field.

**Comment by Mas4hmad** [comment]
Thanks, i just finished. please review again

**Comment by learn-more** [comment]
> You can just set your full name `Mas Ahmad Muhammad` in your [profile settings](https://github.com/settings/profile). Once PR is merged, it will have correct author field.

Only when commits are squashed?

**Comment by binarymaster** [comment]
> Only when commits are squashed?

Yes.

File: base/applications/cmdutils/label/lang/id-ID.rc [hunk 1]
```diff
@@ -1,20 +1,20 @@
-/* Indonesian language file by Zaenal Mutaqin <ade999 at gmail dot com> 2007-02-15 */
+/* Indonesian language file by Zaenal Mutaqin <ade999 at gmail dot com> 2007-02-15 and mas ahmad muhammad <m dot charly81 at gmail dot com> */
```
Comment (HBelusca) [inline base/applications/cmdutils/label/lang/id-ID.rc:1]:
```suggestion
/* Indonesian language file by Zaenal Mutaqin <ade999 at gmail dot com> 2007-02-15 and Mas Ahmad Muhammad <m dot charly81 at gmail dot com> */
```

File: base/system/regsvr32/lang/id-ID.rc [hunk 2]
```diff
@@ -1,21 +1,21 @@
-/* Indonesian translation by Zaenal Mutaqin 2007-02-15 */
+/* Indonesian translation by Zaenal Mutaqin 2007-02-15 an Mas Ahmad Muhammad 2020-04-07*/
```
Comment (HBelusca) [inline base/system/regsvr32/lang/id-ID.rc:1]:
```suggestion
/* Indonesian translation by Zaenal Mutaqin 2007-02-15 and Mas Ahmad Muhammad 2020-04-07 */
```

**Pull request merged**